packages/react-notifications/jest.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.(ts|tsx)'],
+  // ignore coverage for top level "export", version and style files
+  coveragePathIgnorePatterns: ['<rootDir>/src/(index|styles|version).(ts|tsx)'],
```

Comment:
There is no version file or _.tsx_ files that we need to ignore right?

Reply:
yes, will fix it now

Reply:
Pretty sure i left this feedback on an earlier PR as well. Is there any potential that other changes got dropped?

Reply:
That feedback was on Geo PR.
Forgot to make that change here as well.